### ISSUE TITLE
cycle2.cover_theme.css cannot be overriden

### ISSUE THREAD

**Issue by fredvd** [collective/covertile.cycle2#17]
The extra cycle2 css is now defined in way that cannot be overriden in Plone. The file is in a resourceDirectory, we should change this to a separate named ++resource++. 

(I first thought there was an issue with the double dots in the name, but double dots work fine with z3c.jbot if they are for example old style css files in portal_skins. )

**Comment by hvelarde** [comment]
+1

**Comment by djowett** [comment]
Yep, I'd agree with that too, if you don't mind doing it.  It's probably a good point to replace the first "." in the  filename as well, hopefully that will reduce potential confusion.
